pkg/kubelet/cri/remote/remote_runtime_test.go

```diff
@@ -55,6 +60,18 @@ func createRemoteRuntimeService(endpoint string, t *testing.T) internalapi.Runti
 	return runtimeService
 }
 
+func createRemoteRuntimeServiceWithTracerProvider(endpoint string, t *testing.T) internalapi.RuntimeService {
+	ep := "localhost:4317"
+	sr := int32(99999)
+	traceConfig := &tracingapi.TracingConfiguration{Endpoint: &ep, SamplingRatePerMillion: &sr}
+	tp, err := tracing.NewProvider(context.Background(), traceConfig, []otlpgrpc.Option{}, []otelsdkresource.Option{})
```

Comment:
This is good for testing the config-providing code, but not as helpful for testing that requests generate spans.  Could we instead use the [InMemoryExporter](https://github.com/open-telemetry/opentelemetry-go/blob/main/sdk/trace/tracetest/exporter.go#L47) to check that it is generating a span for the version call?

https://github.com/open-telemetry/opentelemetry-go/blob/1eae91b3b099af62100cd630cd1cab89fe841615/sdk/trace/batch_span_processor_test.go#L575 is an example test you can model it after.

Reply:
added a new test to verify spans are exported with version call